students/rgpag/lesson02/series.py

```diff
@@ -0,0 +1,72 @@
+def fibonacci(n):
+	"""fibonacci(n) - return "n"th value of the fibonacci series"""
+	'''fibL1=1
+	fibL2=0
+	if n<0:
+		print("please enter positive int value")
+
+	elif n==0:
+		return 0
+
+	elif n==1:
+		return 1
+
+	else:
+		for i in range(n-1):
+			fibC=fibL1+fibL2
+			fibL2=fibL1
+			fibL1=fibC
+		return fibC
+		'''
```

Comment:
It's a good idea to remove commented code before you commit. If you commit each small change to your code, you can always go back and look at the older version if you want to